src/components/Cards/Results.vue

```diff
@@ -0,0 +1,326 @@
+              </p>
+            </div>
+            <div class="flex-col items-center">
+              <div
+                class="
+                  rounded-lg
+                  px-4
+                  md:px-8
+                  py-1
+                  text-xs
+                  md:text-sm
+                  font-semibold
+                "
+                :class="toggleColor"
+              >
+                Total Product Price
+              </div>
+              <p class="amount font-bold text-lg text-center">
+                {{ downpaymentResult(biztype, selectedDownpayment.total) }}
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <!-- desktop view -->
+    <div
+      class="hidden px-16 py-8 mb-8 bg-white rounded-lg lg:flex flex-col"
+      v-for="(b_type, index) in businessTypes"
+      :key="index"
+    >
+      <div>
+        <p class="font-black text-2xl tracking-wid leading-3 mb-4">
+          {{ b_type.name }}
+        </p>
+        <table class="flex flex-col justify-center items-evenly">
+          <thead
+            class="my-2 px-3 pt-4 flex justify-evenly items-center space-x-10"
+          >
+            <td class="flex-grow pl-24"></td>
+            <td class="font-bold text-lg">Initial Payment</td>
+            <td class="font-bold text-lg">Bi-Monthly Repayment</td>
+            <td class="font-bold text-lg">Total Repayment</td>
+            <td class="font-bold text-lg">Total Product Price</td>
+          </thead>
+          <tbody v-for="(downpayments, index) in computedGetCalc" :key="index">
+            <tr
+              class="my-2 px-3 py-2 flex justify-evenly items-center space-x-20"
+              v-if="downpayments.bizId == b_type.id"
+              :class="toggleDownpaymentColor(downpayments)"
+            >
+              <td class="font-bold text-lg -mr-6 -ml-10">
+                {{ downpayments.percent }}%
+              </td>
+              <td class="font-medium text-base">
+                {{
+                  $formatCurrency(downpayments.actualDownpayment)
+                }}
+              </td>
+              <td class="font-medium text-base">
+                {{
+                  $formatCurrency(downpayments.biMonthlyRepayment)
+                }}
+              </td>
+              <td class="font-medium text-base">
+                {{
+                  $formatCurrency(downpayments.actualRepayment)
+                }}
+              </td>
+              <td class="font-medium text-base">
+                {{
+                   $formatCurrency(downpayments.total)
+                }}
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      id: this.$route.params.name,
+    };
+  },
+  computed: {
+    toggleColor() {
+      let color = "";
+      let selectDownpayment = !this.selectedDownpayment
```

Comment:
change this to` let selectDownpayment = this.selectedDownpayment?.percent`

Reply:
done